smart-contracts/package.json

```diff
@@ -53,6 +53,7 @@
     "lint": "yarn lint:contracts && yarn lint:code",
     "lintFix": "yarn lint:contracts --fix --noPrompt && yarn lint:code --fix",
     "coverage": "IS_COVERAGE=1 yarn hardhat coverage",
+    "coverage_fork": "IS_COVERAGE=1 RUN_FORK=1 yarn hardhat coverage",
```

Comment:
can you run only the test with prefix `*.mainnet.js` here please ?

Reply:
@clement, I have updated the script.